### DISCUSSION TITLE
how to install?

### DISCUSSION THREAD

**Discussion by Penfeld29** [elementary/os#672] [Q&A] [answered]
I can mount the iso file on my computer. But then what to do ?
what is the file to click on ? I've tried the EFI file but I got an error message telling my HD must be in NTFS. (My HD is in HTFS)
Can you help?
![Capture](https://github.com/elementary/os/assets/137454395/c3780eee-4ac1-4641-8c0a-21ca62758189)

**Comment by gamerlv** [comment, marked as answer]
elementary os is an replacement for Windows and macOS. It can not be installed from within Windows.

There is a full guide on the website https://elementary.io/docs/installation ([fr](https://elementary.io/fr/docs/installation)) with steps how to install and get started with using elementary os.

**Reply by Penfeld29** [reply]
Thanks a lot.
